go.mod

```diff
@@ -1,5 +1,5 @@
 module github.com/adobe/aquarium-fish
-
+# diff needed
```

Comment:
working, this will launch CI and then follow-up commit with the actual change after results to compare

Reply:
[CodeQL results](https://github.com/adobe/aquarium-fish/actions/runs/9103247836/job/25024687533?pr=68#step:4:2313) from this PR via https://github.com/adobe/aquarium-fish/pull/68/commits/8c7da5f1be7c753f4bdaa94441bf886a759b5f86

Reply:
Success :partying_face:  There is no `Invalid Go toolchain version` section in the [new CodeQL results](https://github.com/adobe/aquarium-fish/actions/runs/9103290224/job/25024822914?pr=68#step:4:2310).

Reply:
@sparshev I'm going to leave this review "unresolved", to make the review process for this PR a little easier.  The change is pretty minor, but the only thing that reports it is the CodeQL CI check.  If it looks good to you we'll resolve it :+1: